.gitignore

```diff
@@ -86,4 +86,5 @@ Icon
 .AppleDesktop
 Network Trash Folder
 Temporary Items
-.apdisk
\ No newline at end of file
+.apdisk
+.vscode/
```

Comment:
What about using your [global gitignore](https://help.github.com/pt/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) for editor files? As there are several editors and new ones are created, this behavior avoid a ripple effect of changes when people adopt different editors.

Reply:
:+1: